sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Crop Box Filter Node",
+  "type": "object",
+  "definitions": {
+    "crop_box_filter": {
+      "type": "object",
+      "properties": {
+        "min_x": {
+          "type": "number",
+          "description": "x-coordinate minimum value for crop range",
```

Comment:
```suggestion
          "description": "minimum x-coordinate value for crop range",
```

Reply:
Applicable to other instances. Units

Reply:
Thanks! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8466/commits/772c90251b465caeab41535c50fc12f39ab10a32